OpenRA.Game/Network/Handshake.cs

```diff
@@ -77,10 +77,12 @@ public static HandshakeResponse Deserialize(string data)
 
 		public string Serialize()
 		{
-			var data = new List<MiniYamlNode>();
-			data.Add(new MiniYamlNode("Handshake", null,
-				new[] { "Mod", "Version", "Password", "Fingerprint", "AuthSignature", "OrdersProtocol" }.Select(p => FieldSaver.SaveField(this, p)).ToList()));
-			data.Add(new MiniYamlNode("Client", FieldSaver.Save(Client)));
+			var data = new List<MiniYamlNode>
+			{
+				new MiniYamlNode("Handshake", null,
+				new[] { "Mod", "Version", "Password", "Fingerprint", "AuthSignature", "OrdersProtocol" }.Select(p => FieldSaver.SaveField(this, p)).ToList()),
```

Comment:
```suggestion
				new MiniYamlNode("Handshake", null,
					new[] { "Mod", "Version", "Password", "Fingerprint", "AuthSignature", "OrdersProtocol" }.Select(p => FieldSaver.SaveField(this, p)).ToList()),
```
(Commit " Simplify collection initialization")